lib/fetch/management-kontent-fetcher.ts

```diff
@@ -0,0 +1,101 @@
+import { HttpService } from '@kontent-ai/core-sdk';
+import {
+    AssetFolderModels,
+    CollectionModels,
+    ContentItemModels,
+    ContentTypeModels,
+    ContentTypeSnippetModels,
+    createManagementClient,
+    EnvironmentModels,
+    LanguageModels,
+    RoleModels,
+    TaxonomyModels,
+    WebhookModels,
+    WorkflowModels
+} from '@kontent-ai/management-sdk';
+import chalk from 'chalk';
+import { toSafeComment } from '../core/comment.utils.js';
+import { GeneratorManagementClient } from '../core/core.models.js';
+
+interface KontentFetcherConfig {
+    readonly environmentId: string;
+    readonly apiKey: string;
+    readonly baseUrl?: string;
+}
+
+export function managementKontentFetcher(config: KontentFetcherConfig) {
+    const client: GeneratorManagementClient = createManagementClient({
+        environmentId: config.environmentId,
+        apiKey: config.apiKey,
+        baseUrl: config.baseUrl,
+        httpService: new HttpService({ logErrorsToConsole: false })
```

Comment:
What kind of header do you have in mind? There is the `X-KC-SDKID`, which would already be used by underlying packages (i.e. management SDK). Is there some specific header I could use? If so, I'll be happy to use it ;)

Reply:
The `X-KC-SDKID` defines the SDK used, right? So we wouldn't be able to differentiate between traffic from the model generator and a user app. We use `X-KC-SOURCE` to track the traffic of different tools. We use this format `${packageJson.name};${packageJson.version}`.

Reply:
Yeah, that's why I was asking if there is some other header I could use :) I haven't used the `X-KC-SOURCE` header yet, but I knew I heard it somewhere :) 

Looking at the code I think I'll have to expose a new config in the `ManagementClient` so that we can set this header there, rather than in every query config.

Reply:
I've added the header & updated Management SDK to allow global configuration of custom headers.